scripts/developer_tools/j_commit_and_push.py

```
@@ -0,0 +1,198 @@
+        )
+        return result.stdout.strip()
+    except subprocess.CalledProcessError as exc:
+        print(f"ERROR: Not a git repository or git command failed: {exc}", file=sys.stderr)
+        raise SystemExit(1) from exc
+
+
+def stage_changes(files: Optional[list[str]]) -> None:
+    """Stage the given files, or all changes (tracked and untracked) if none given."""
+    if files:
+        for f in files:
+            subprocess.run(["git", "add", f], check=True)
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)